scss/skin/_default.scss

```diff
@@ -47,12 +47,14 @@
 
   // "full window" style, if fullscreen is not supported natively
   &.oo-fullscreen {
-    position:fixed;
-    overflow: hidden;
-    top:0;
-    left:0;
-    width:100%;
-    height:100%;
+    width: 100%!important;
```

Comment:
Please check to see if `!important` is really necessary in these styles. If not, please remove.

Reply:
we refactored css